cfme/configure/settings.py

```diff
@@ -166,4 +187,34 @@ def login_page(self, value):
         fill(self.startpage_form.login_page, str(value))
         sel.click(form_buttons.save)
 
+    def update(self, updates):
+        sel.force_navigate("my_settings_visual", context=self)
+        fill(self.quadicons_form, {'infra_provider_quad': updates.get('infra_provider_quad'),
+                                   'cloud_provider_quad': updates.get('cloud_provider_quad'),
+                                   'host_quad': updates.get('host_quad'),
+                                   'datastore_quad': updates.get('datastore_quad'),
+                                   'datastoreitem_quad': updates.get('datastoreitem_quad'),
+                                   'vm_quad': updates.get('vm_quad'),
+                                   'vmitem_quad': updates.get('vmitem_quad'),
+                                   'template_quad': updates.get('template_quad')})
+        sel.click(form_buttons.save)
+
+    def uncheck_checkboxes(self):
+        sel.force_navigate("my_settings_visual")
+        fill(self.quadicons_form, {'infra_provider_quad': self.infra_provider_quad,
```

Comment:
If this really is supposed to uncheck all the checkboxes, shouldn't all of the values be `False`?